Makefile.am

```diff
@@ -58,7 +58,7 @@ all_platform_headers += $(linux_platform_headers)
 
 if HTOP_LINUX
 AM_LDFLAGS += -rdynamic
-myhtopplatsources = linux/Platform.c linux/IOPriorityPanel.c linux/IOPriority.c \
+myhtopplatsources = linux/Platform.c linux/IOPriorityPanel.c \
```

Comment:
Why was `linux/IOPriority.c` removed?

Reply:
Because aside from being a source from which the corresponding header was generated, it included nothing of actual substance -- an enum declaration, a few macros, and a typedef (zero variable or function definitions).

Reply:
sorry, missed, that the `.c` file was removed in reallity too.